model/lib/email/email_content.dart

```diff
@@ -12,4 +12,19 @@ class EmailContent with EquatableMixin {
 
   @override
   List<Object?> get props => [type, content];
+}
+
+extension EmailContentExtension on EmailContent {
+  String get asHtml {
+    if (type == EmailContentType.textPlain) {
+      return content
+          .replaceAll("'", r"\'")
+          .replaceAll('"', r'\"')
+          .replaceAll('\r', '')
+          .replaceAll('\r\n', '')
+          .replaceAll('\n', '<br/>')
+          .replaceAll('\n\n', '<br/>');
+    }
+    return content;
+  }
```

Comment:
I'd prefer to rely on a library to do text => HTML because now we are vulnerable to HTML injections

Reply:
#309

Reply:
Hi @chibenwa, back to that problem, we found that no library support us on that (build up the block quote html from plaintext). 
So how about we still keep our method like that, then sanitize the html before rendering?

Reply:
please continue discuss on https://github.com/linagora/tmail-flutter/issues/309#issuecomment-1062698198